examples/network/imageSelected/imageSelected.html

```diff
@@ -0,0 +1,82 @@
+<html>
+<head>
+  <title>Network | Selected/Unselected Image</title>
+
+  <script type="text/javascript" src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis-network.min.css" rel="stylesheet" type="text/css" />
+
+  <style type="text/css">
+    body {
+      font: 10pt arial;
+    }
+    #mynetwork {
+      width: 600px;
+      height: 600px;
+      border: 1px solid lightgray;
+    }
+  </style>
+</head>
+
+<body>
+<div id="mynetwork"></div>
+<script type="text/javascript">
+  // create an array with nodes
+  var nodes = new vis.DataSet([{
+    id: 1,
+    shape: 'image',
+    size: 20,
+    label: 'No select image',
+    image: './unselected.svg',
+  }, {
+    id: 2,
+    shape: 'image',
+    size: 20,
+    label: 'Select image broken',
+    image: {
+      unselected: './unselected.svg',
+      selected: './BROKEN_LINK/selected.svg',
+    },
```

Comment:
You decide :) I like to use trailing commas as part of my code style because all lines look the same and you can safely copy-paste them. Really it's up to you, depending on the code style of vis.js.